README.md

```diff
@@ -2,6 +2,16 @@
 
 Another Kubernetes Helper for shipping code.
 
+## Dependencies
+Ankh uses kubectl and helm under the hood, make sure you install them.
+Kubectl: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+Helm: https://github.com/kubernetes/helm
+
+## Build and Installation
+To build `ankh`, just run `make`
+
+To install `ankh` simply run `make install`.
```

Comment:
we can keep the language consistent:

"..., run make"

and

"..., run make install"